tests/project/test_projectPath.py

```diff
@@ -67,18 +68,18 @@ def test_path(self):
         )
 
     def test_root_path(self):
-        root_paths = self.settings_configuration["project_paths"]
+        root_paths = state.settings.configuration["project_paths"]
         self.assertIn(
-            self.project_path.root_path,
+            self.project_path.root_path[:-1],  # Trim the trailing "/"
```

Comment:
If there is no tailing `/` then this fails right? So I m not sure if that improves the issue ...

Reply:
On my local machine all that was needed was to remove the trailing /. Now It's failing only for windows, so I think this was almost the right move....

Reply:
Ah, no, on windows it looks like the issue was actually with the 'join' command which inserted `//` while pyiron stores everything as unix.

Reply:
> On my local machine all that was needed was to remove the trailing /. Now It's failing only for windows, so I think this was almost the right move....

The tailing `/` is just what you defined in your `.pyiron` file, right?

Reply:
This doesn't rely on the config file any more, but on explicit updates in the test setup:

```python
    def setUp(self) -> None:
        state.settings.configuration["project_paths"] = [self.current_dir]
        state.settings.configuration["project_check_enabled"] = True

        self.project_path = ProjectPath(path=self.current_dir)
        self.project_path = self.project_path.open("test_project_path")
```

`ProjectPath` must be appending the `/` to `self.current_dir`, but give me a minute to find where.

(btw `self.current_dir` is just a unix-ized path in the class setup:
```python
@classmethod
    def setUpClass(cls):
        if os.name == "nt":
            cls.current_dir = os.path.dirname(os.path.abspath(__file__)).replace(
                "\\", "/"
            )
        else:
            cls.current_dir = os.path.dirname(os.path.abspath(__file__))
```)

Reply:
Yeah, `ProjectPath.__init__` calls `ProjectPath._convert_str_to_generic_path` which calls `GenericPath._windows_path_to_unix_path`:

```python
        if path is not None:
            linux_path = path.replace("\\", "/")
            if linux_path[-1] != "/":
                linux_path += "/"
            return linux_path
        else:
            return None
```

Reply:
From my perspective the issue is that `state.settings.configuration["project_paths"]` depends on the users `.pyiron` config or environment variables. So while some users have their paths ending with `/` others do not. So I would recommend cleaning up the `state.settings.configuration["project_paths"]` used in the tests, rather than modifying the tests because the whole `[:-1]` notation to remove the tailing `/` is very confusing.

Reply:
Yep, sounds good to me.